bodhi/server/buildsys.py

```diff
@@ -448,7 +449,7 @@ def getTag(self, taginfo, **kw):
                 'perm': None, 'id': 246, 'arches': None,
                 'maven_include_all': False, 'perm_id': None}
 
-    def addTag(self, tag: str, **opts):
+    def createTag(self, tag: str, **opts):
```

Comment:
I think we still want `addTag` here. I would have assumed that this is used in some test.

Reply:
It's not, I checked the whole project before doing this change.
I did it, because the koji API doesn't have any method `addTag`.